carsblog/cars/migrations/0001_initial.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 3.2.5 on 2021-07-16 19:16
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Car',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('brand', models.CharField(max_length=30)),
+                ('model', models.CharField(max_length=30)),
+                ('generation', models.CharField(max_length=30)),
+                ('year_of_prod', models.DateField()),
+                ('displacement', models.IntegerField()),
+                ('power', models.IntegerField()),
+                ('body_type', models.CharField(max_length=30)),
+                ('fuel_type', models.CharField(max_length=30)),
+                ('description', models.TextField()),
+            ],
+        ),
+    ]
```

Comment:
Remove migration file from repo. Add in _.gitignore_ file rule to ignore all python migration files in directory _carsblog/cars/migrations/_ (**but this directory should be visible in repo**).